src/PermissionServiceProvider.php

```diff
@@ -86,6 +90,19 @@ protected function registerCommands(): void
         ]);
     }
 
+    protected function registerOctaneListener(): void
+    {
+        if ($this->app->runningInConsole() || ! $this->app['config']->get('octane.listeners')) {
+            return;
+        }
+
+        $dispatcher = $this->app[Dispatcher::class];
+
+        $dispatcher->listen(\Laravel\Octane\Events\RequestReceived::class, OctaneReloadPermissions::class);
```

Comment:
Perhaps we should just listen on `OperationTerminated`?

> The `OperationTerminated` event is a interface that is implemented by `TickTerminated`, `TaskTerminated` and `RequestTerminated`, which all get fired depending on the context of the application (receiving a request or executing a job/command).

Reply:
Done

Reply:
Thanks